### PR TITLE
Remove all the files from the master branch except README.md

### PR DESCRIPTION
**Pull request by ismail-shahzad**
Removes all the files and updates the README.md with links to the new dedicated branches.
Fixes https://gitlab.com/datopian/tech/devops/-/issues/118
The master branch is now only going to contain links to the dedicated branches of the projects. Links to the projects can be added or removed from the README.md as required.

The README.md now looks like this
Screenshot from 2020-08-31 19-37-26

### CODE REVIEW

File: .docker-compose-cache-from.yaml [hunk 1]
```diff
@@ -1,33 +0,0 @@
-version: '3.2'
```
Comment (zelima) [inline .docker-compose-cache-from.yaml:1]: Why are you deleting this file?

Comment (zelima) [reply .docker-compose-cache-from.yaml:1]: Worst case we should make this generic. Example kind of...

File: .docker-compose-centralized.yaml [hunk 2]
```diff
@@ -1,27 +0,0 @@
-version: '3.2'
```
Comment (zelima) [inline .docker-compose-centralized.yaml:1]: Same - don't delete but make it generic.

File: .gitignore [hunk 3]
```diff
@@ -1,117 +0,0 @@
-# Byte-compiled / optimized / DLL files
```
Comment (zelima) [inline .gitignore:1]: Why deleting this file?

File: .travis.sh [hunk 4]
```diff
@@ -1,40 +0,0 @@
-#!/usr/bin/env bash
```
Comment (zelima) [inline .travis.sh:1]: same - Why deleting this file?

File: .travis-ci-operator.yaml [hunk 5]
```diff
@@ -1,5 +0,0 @@
-encryptedDockerPassword: KNKc72qckJRJxSLzxPEWDGKUSIlUvoOWOdtQopoie9MGv9VUjvzL5kwqOkNkYN5ONZU8RIzVRHywf787zC/qjdYoAOP4ZlDOTHszmIRj39uBAqu3qfri+KAqqG/mkt5xeBa0qNJHphmqEDofsplaDh9xkssvWvbABl8gJ8PyOzTwM5SR2lFW/kHzwyspxhj/l6tjZpYbOQeXQllBabRRzI4j2BRQbz+12cMM6doGR29V3yuGmjbSU5uXful5tSlKLfYqQ4Rf7g00mirLO3I2fKuRFZAqIRNpbV6dLIpSQc6xRnWvGQ2Rpkec3Y9O3dXMwVZyR2KaLHApg9VBDhaRy3H81ZJggVNlUYHKyS83E8oGf42BV9U5UPmoT5x8IVK3oy4zT63z6l5nLyGvf31DH/cKYanF1e9QwxXBPp18de8iJjKsEk76THxKpx8aZIbaniu+TLvBhBT+wvayL/FSqjGp6HS6W6wD8hFod7b9ia4AiTSj7haCAIsK4VouRtIA7LhZTvgABUnhzgeG1ASxYu/pMBj3jIIBib1+G6qfWVRU/Z/Htwm0d/viSncIuSOUb4CbZjgHZOcv5vPrr/IdSsv3dTzebtrDE4OW/iHnzr06yqyPA7TUYcP7QYlXeCpYChYKYUEuN/maaVF31MOGFX3jsQRdApBimcA1KpH3MZQ=
```
Comment (zelima) [inline .travis-ci-operator.yaml:1]: same - Why deleting this file?

**Review by zelima — CHANGES_REQUESTED** [review]
@MuhammadIsmailShahzad so you are deleting all the files? Let' don't do that. Keep all the files (esp travis related). remove files that are related to specific instances like vital-strategies or similar. 

Also, we need one generic example (you can keep data gov one and maybe rename to datopian or similar...)

**Comment by ismail-shahzad** [comment]
WONTFIX. Closing this PR. Will open with fixes implemented in a fresh fork.

**Pull request closed without merging**